### ISSUE TITLE
VSnackbar  Invalid prop: type check failed for prop "text". on vuetify 2.3

### ISSUE THREAD

**Issue by mean-cj** [yariksav/vuetify-dialog#82]
Hello

Today vuetify have releases version 2.3
https://vuetifyjs.com/en/getting-started/releases-and-migrations/

I found problem with VSnackbar 

```
[Vue warn]: Invalid prop: type check failed for prop "text".
Expected Boolean, got String with value "MESSAGE".

found in

---> <VSnackbar>
       <Root>
```

Thank you for your project.

<details><summary><b>Logs</b>:</summary>

warn | @ | vue.runtime.esm.js?2b0e:619
  | assertProp | @ | vue.runtime.esm.js?2b0e:1705
  | validateProp | @ | vue.runtime.esm.js?2b0e:1632
  | loop | @ | vue.runtime.esm.js?2b0e:4665
  | initProps | @ | vue.runtime.esm.js?2b0e:4698
  | initState | @ | vue.runtime.esm.js?2b0e:4639
  | Vue._init | @ | vue.runtime.esm.js?2b0e:5006
  | VueComponent | @ | vue.runtime.esm.js?2b0e:5154
  | createComponentInstanceForVnode | @ | vue.runtime.esm.js?2b0e:3283
  | init | @ | vue.runtime.esm.js?2b0e:3114
  | createComponent | @ | vue.runtime.esm.js?2b0e:5978
  | createElm | @ | vue.runtime.esm.js?2b0e:5925
  | createChildren | @ | vue.runtime.esm.js?2b0e:6053
  | createElm | @ | vue.runtime.esm.js?2b0e:5954
  | patch | @ | vue.runtime.esm.js?2b0e:6477
  | Vue._update | @ | vue.runtime.esm.js?2b0e:3945
  | updateComponent | @ | vue.runtime.esm.js?2b0e:4060
  | get | @ | vue.runtime.esm.js?2b0e:4479
  | Watcher | @ | vue.runtime.esm.js?2b0e:4468
  | mountComponent | @ | vue.runtime.esm.js?2b0e:4073
  | Vue.$mount | @ | vue.runtime.esm.js?2b0e:8415
  | show | @ | vuetify-dialog.esm.js?83df:518
  | show | @ | vuetify-dialog.esm.js?83df:753
  | eval | @ | vuetify-dialog.esm.js?83df:770
  | success | @ | vuetify-dialog.esm.js?83df:2471
  | eval | @ | axios.js?dcce:140
  | eval | @ | axios.js?10e8:26
  | Promise.then (async) |   |  
  | request | @ | Axios.js?0a06:61
  | wrap | @ | bind.js?1d2b:9
  | eval | @ | index.js?25e2:415
  | submit | @ | index.js?25e2:414
  | post | @ | index.js?25e2:333
  | onSubmit | @ | location.vue?deac:225
  | submit | @ | location.vue?8fd4:25
  | invokeWithErrorHandling | @ | vue.runtime.esm.js?2b0e:1854
  | invoker | @ | vue.runtime.esm.js?2b0e:2179
  | invokeWithErrorHandling | @ | vue.runtime.esm.js?2b0e:1854
  | Vue.$emit | @ | vue.runtime.esm.js?2b0e:3888
  | submit | @ | vuetify.js?ce5b:15666
  | invokeWithErrorHandling | @ | vue.runtime.esm.js?2b0e:1854
  | invoker | @ | vue.runtime.esm.js?2b0e:2179
  | original._wrapper | @ | vue.runtime.esm.js?2b0e:6917

</details>

**Comment by yariksav** [comment]
fixed in e9bf142990ecca1c701ce31031dfd7197378bda8
please use vuetify-dialog@2.0.4

**Comment by mean-cj** [comment]
Hello @yariksav 

after update i will found warning on dev 

console.ts?522c:33 [Vuetify] Unable to locate target [data-app]

found in

---> <VDialog>
       <Root>

  | consoleWarn | @ | console.ts?522c:33
-- | -- | -- | --
  | initDetach | @ | index.ts?3875:142
  | mounted | @ | index.ts?3875:82
  | invokeWithErrorHandling | @ | vue.runtime.esm.js?2b0e:1854
  | callHook | @ | vue.runtime.esm.js?2b0e:4219
  | insert | @ | vue.runtime.esm.js?2b0e:3139
  | invokeInsertHook | @ | vue.runtime.esm.js?2b0e:6346
  | patch | @ | vue.runtime.esm.js?2b0e:6565
  | Vue._update | @ | vue.runtime.esm.js?2b0e:3945
  | updateComponent | @ | vue.runtime.esm.js?2b0e:4060
  | get | @ | vue.runtime.esm.js?2b0e:4479
  | Watcher | @ | vue.runtime.esm.js?2b0e:4468
  | mountComponent | @ | vue.runtime.esm.js?2b0e:4073
  | Vue.$mount | @ | vue.runtime.esm.js?2b0e:8415
  | show | @ | vuetify-dialog.esm.js?83df:518
  | show | @ | vuetify-dialog.esm.js?83df:754
  | eval | @ | vuetify-dialog.esm.js?83df:771
  | eval | @ | auth.firebase.js?360e:75
  | Promise.catch (async)



Thank you for your fixed
